### DISCUSSION TITLE
Node.js project bump package.json

### DISCUSSION THREAD

**Discussion by RWOverdijk** [orhun/git-cliff#953] [Q&A] [answered]
Hi!

So far I really enjoy using this project so far and I was wondering if it's also possible to bump the version in package.json when using --bump. Right now I use --tag with a separate npm version. This works, but it would be nice if I could combine those into 1 call.

Just in case this is already possible, I thought I'd ask.

**Comment by orhun** [comment, marked as answer]
Hey! 👋🏼 

This is currently not possible and I don't think I will be implementing it, it is out of scope of `git-cliff`. You can get the bumped version via `--bumped-version` and use as you like though :)

**Reply by RWOverdijk** [reply]
I figured that would be the case, it makes most sense. It's what I do now. Just `NEXT_VER=$(npx git-cliff@latest --bumped-version --unreleased)` and toss that bad boy in all the right places.

Thanks for the reply and have a nice day :)
